service/src/main/java/uk/gov/hmcts/reform/fpl/utils/ControllerHelper.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+public class ControllerHelper {
```

Comment:
I am not sure if the ControllerHelper is good name I mean we process something on caseDetails, the reference to controller seems confusing.